packages/sql-mapper/lib/telemetry.js

```diff
@@ -33,7 +29,7 @@ function wrapQuery (app, db, request) {
 
     const ctx = request.span?.context
 
-    const { database, host, port, user } = connectionInfo
+    const { database, host, port, user, dbSystem } = connectionInfo
```

Comment:
are those changes needed? Can you add a test for them?

Reply:
This is a small refactory (which yes, is not strictly necessary) so it's already covered by tests (e.g.: https://github.com/platformatic/platformatic/blob/connection-stirngs-management-api/packages/sql-mapper/test/telemetry.test.js#L116).